README.md

```diff
@@ -883,10 +883,15 @@ This is due to already loaded libraries, for more information see this [blog pos
 
 Release 1.8 implemented an extension point that will workaround this issue. In order to work a new task has to be created in the Windows Task Scheduler named [RESTART_JENKINS_AGENT](docs/RESTART_JENKINS_AGENT.xml) and configured with actions how to restart the agent.
 
-Please not that from [ECU-TEST](https://www.tracetronic.de/produkte/ecu-test/) Version 2022.3 on, **invalid escape sequences in expressions within ECU-TEST are 
+Please note that from [ECU-TEST](https://www.tracetronic.de/produkte/ecu-test/) Version 2022.3 on, **invalid escape 
+sequences in expressions within ECU-TEST are 
 disallowed.** This means, in particular, that backslashes in Windows paths need to be escaped properly (e.g. 
 *C:\\\my\\\path* instead of *C:\my\path*) - otherwise, this will lead to an error within ECU-TEST.
 
+Since release 1.39 we test only against [ECU-TEST](https://www.tracetronic.de/produkte/ecu-test/) versions, that are 
```

Comment:
* please link the [release version 1.39](https://github.com/jenkinsci/ecutest-plugin/releases/tag/ecutest-2.39) 
* specify 'the  support terms' as 'TraceTronic tool support terms'
* since we have the[ english version](https://www.tracetronic.com/products/ecu-test/) available we should use it